pkg/agent/const.go

```diff
@@ -105,6 +105,7 @@ const (
 	updateNodeLabelsSystemdService            = "linux/cloud-init/artifacts/update-node-labels.service"
 	kubernetesCustomSearchDomainsScript       = "linux/cloud-init/artifacts/setup-custom-search-domains.sh"
 	kubeletSystemdService                     = "linux/cloud-init/artifacts/kubelet.service"
+	krustletSystemdService                    = "linux/cloud-init/artifacts/krustlet.service"
```

Comment:
krustletFixCaScript needs to be added here

Reply:
https://github.com/Azure/AgentBaker/blob/304ad9410059f42721dcb0f2f68651dc18fb1dd0/pkg/agent/variables.go#L29

krustletFixCa and krustletSystemdService need to be added here too